src/PhpPact/Standalone/PactMessage/PactMessage.php

```diff
@@ -41,9 +41,9 @@ public function reify(Message $pact): string
         $scripts = $this->installManager->install();
 
         $json    = \json_encode($pact);
-        $process = new AsyncProcessRunner($scripts->getPactMessage(), ['reify', "'" . $json . "'"]);
+        $process = new ProcessRunner($scripts->getPactMessage(), ['reify', "'" . $json . "'"]);
 
-        $process->run();
+        $process->run($blocking = true);
```

Comment:
The result of the assignment will be sent as an argument to the method call. I use $blocking = false because I find it more readable, a workaround for PHP not having named parameters.
However, I don't like boolean method arguments anyway. Let me think of an alternative implementation